app/src/main/java/com/khuttun/notificationnotes/NotificationMgr.java

```diff
@@ -65,18 +67,11 @@ public void setNotification(NotificationNote note)
 
     public void setGroupNotification(ArrayList<NotificationNote> notes)
     {
-        ArrayList<NotificationNote> visibleNotes = new ArrayList<>();
-        for (int i = 0; i < notes.size(); ++i)
-        {
-            NotificationNote n = notes.get(i);
-            if (n.isVisible)
-            {
-                visibleNotes.add(n);
-            }
-        }
-
+        clearAllNotifications();
```

Comment:
Calling clearAllNotifications() here doesn't seem right. Would it make more sense to only call this when setting the Nougat+ group notifications, if this is needed for them?

Reply:
The function `clearAllNotifications` is now called in `displayGroupNotifications`.